pysatModels/utils/extract.py

```diff
@@ -15,6 +15,73 @@
 import pysatModels as ps_mod
 
 
+def compare_mod_name_coordinates(data, mod_name):
```

Comment:
Recommend adding this to the test class it is used in.

Reply:
Just checking that you decided not to move this?  Given the thumbs up wasn't sure if you forgot or decided not to.